milestone-3/frontend/src/App.js

```diff
@@ -0,0 +1,58 @@
+import './App.css';
+import recipes from './components/data';
+import Title from './components/begin';
+import Recipe from './components/Recipes';
+import Navbar from './components/navbar';
+import React, { useState, useState} from "react";
+import { Route, Routes, BrowserRouter} from 'react-router-dom';
+
+function App() {
+  let [recipes, setRecipes] = useState();
+  useEffect(() => {
+    const loadAllRecipes = async () => {
+      fetch('http://localhost:3001/api/recipe')
+      .then(res => res.json())
+      .then(jsondata => setRecipes(jsondata))
+      .catch(err => console.log(err))
+    }
+  loadAllRecipes();
+}, []);
+
+  return(
+    <BrowserRouter>
+      <div className = "App">
+        {recipes ? (
+          <div> 
+            <Navbar />
+            <div className = "pageData"> 
+              <Routes>
```

Comment:
I believe this should be Switch instead of Routes
see: 
https://www.notion.so/Using-React-Router-c1365e4e3d9e46c8b04325063ce63da5